src/classes/RecipeRepository.js

```diff
@@ -1,34 +1,90 @@
 
 
 class RecipeRepository {
-  constructor(data) {
-    this.recipeData = data;
-  }
+  constructor(recipeData, ingData) {
+    this.recipeData = recipeData;
+    this.ingredientData = ingData;
+  };
 
-  filterByNameOrIng(nameOrIng, ingArray) {
-  // check for ingredient:
-  console.log(ingArray)
-  //function that searches ing array and returns array of ids
-  let getIDfromName = ingArray.filter((element) => {
-      return element.name.includes(nameOrIng)
-    });
-  //call function from above and store arry in var idList
-  let idList = getIDfromName().map((element) => {
-    return element.id
-  })
+  filterByNameOrIng(nameOrIng) {
+    // 1. create empty array that will hold recipes we want to display (from search)
```

Comment:
Really happy with how much we're psuedocoding to be able to SEE our thought processes.